shared/types.ts

```diff
@@ -11,16 +11,27 @@ export enum VariableModFunction {
   Set
 }
 
+export enum imageType {
+  string,
+  jpg,
+  png
+}
+
 export interface IUpgradeEffect {
   variableAffected: GameVariable;
   variableMod: VariableModFunction;
   modValue: number;
 }
 
+export interface IUpgradeImage {
```

Comment:
Could we generalize this to be `IAppImage` or something of that nature? We may want to reuse this functionality in other parts of the application.

Reply:
I will rename the interface and make a ticket to expand it's features when needed